SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -15,19 +15,72 @@ \chapter{Configuring and Building ITK}
 
 ITK has been developed and tested across different combinations of operating
 systems, compilers, and hardware platforms including Microsoft Windows, Linux on
-various architectures, Solaris/UNIX, macOS, and Cygwin. Kitware is committed
-to support the following compilers for building ITK:
+various architectures, UNIX, macOS, and Cygwin. Dedicated community members and
+Kitware are committed to providing long-term support of the most prevalent
+development environments (Visual Studio, macOS, and Linux) for building ITK:
+
+Compiler variants will be supported for the duration that the associated operating
+system vendors commit to in their long-term stable platforms. For example the gcc
+compilers supported will mirror the compiler support in the RedHat lifecycle, the
+apple clang compilers will mirror the support lifecycle of the compiler by Apple, and
+the Visual Studio series will have approximately a 10 year support lifecycle.
 
 % List updated according to information provided here:
 % https://www.itk.org/Wiki/ITK_Release_4/Modern_C++#Fully_Committed_to_Support
```

Comment:
I can't remember whether we've talked about deprecating the referenced wiki pages in favour of this file in this case. In case we will be maintaining the list up-to-date only on this file, we should may be remove the reference. Otheriwse, ignore this comment.